### PR TITLE
add some

### PR DESCRIPTION
**Pull request by Illia1927**
create pull request

### CODE REVIEW

File: src/mate/academy/patterns/abstractFactory/train/HyundaiRotem.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public String getType() {
+        return type;
+    }
+
```
Comment (BondarenkoJek) [inline src/mate/academy/patterns/abstractFactory/train/HyundaiRotem.java]: delete line

File: src/mate/academy/patterns/adapter/JavaApp.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    }
+
+    public void loadObject(){
+        System.out.println("loading java object");
+    }
+
+    public void deleteObject(){
+        System.out.println("Deleting java object");
+    }
+
```
Comment (BondarenkoJek) [inline src/mate/academy/patterns/adapter/JavaApp.java]: delete line

File: src/mate/academy/patterns/prototype/Human.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "Human{" +
+                "age : " + age +
+                ", name : " + name + "}";
+    }
+}
+
```
Comment (BondarenkoJek) [inline src/mate/academy/patterns/prototype/Human.java]: delete line

File: src/mate/academy/patterns/prototype/Copyable.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package mate.academy.patterns.prototype;
+
+public interface Copyable {
```
Comment (BondarenkoJek) [inline src/mate/academy/patterns/prototype/Copyable.java:3]: maybe will better use generics?

**Review by BondarenkoJek — CHANGES_REQUESTED** [review]

File: src/mate/academy/patterns/prototype/Copyable.java (hunk 4 above)
Comment (Illia1927) [reply src/mate/academy/patterns/prototype/Copyable.java]: @BondarenkoJek what for?

**Review by BondarenkoJek — APPROVED** [review]
LGTM

**Comment by Illia1927** [comment]
Agon💣

Чт, 14 лют. 2019, 13:40 BondarenkoJek користувач notifications@github.com
пише:

File: src/mate/academy/patterns/adapter/AdapterJavaAppToCppApp.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
+package mate.academy.patterns.adapter;
+
+public class AdapterJavaAppToCppApp implements CppApp {
+    private JavaApp javaApp;
```
Comment (boroda4436) [inline src/mate/academy/patterns/adapter/AdapterJavaAppToCppApp.java:4]: add final variable
```suggestion
    private final JavaApp javaApp;
```

**Review by boroda4436 — APPROVED** [review]
LGTM

**Pull request merged**